projects/core/src/occ/adapters/product/converters/product-image-normalizer.ts

```diff
@@ -98,4 +98,9 @@ export class ProductImageNormalizer implements Converter<Occ.Product, Product> {
         '') + url
     );
   }
+
+  private hasGalleryIndex(image: Occ.Image) {
+    const galleryIndex = image.galleryIndex ?? false;
+    return galleryIndex !== false;
```

Comment:
What do you think about this one liner 

- What you have is fine too though 

```suggestion
    return !isNaN(Number(image.galleryIndex));
```

Reply:
nvm* reverted to your old one